packages/terra-modal-manager/CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ Unreleased
 ------------------
 ### Added
 * Added documentation on required peerDependencies
+* Resolved Merge Conflicts.
```

Comment:
There don't seem to be any changes in this package so this entry isn't needed

Reply:
Resolved in 50d875f